src/styles/card.css

```diff
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  box-sizing: border-box;
```

Comment:
box-sizing should be the same for all elements, use `*` selector for it in normalize
```suggestion
```